RummikubGameApp/RummikubGameApp/OpponentAI.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+
+namespace RummikubGameApp
+{
+    class OpponentAI
+    {
+        private Tile[] AIhand;
+
+        public OpponentAI()
+        {
+            AIhand = new Tile[106];
+        }
+        public void dealtiles()
+        {
+            var instance = new GameLogic();
+            AIhand = instance.DealAI();

```

Comment:
since this is a new class won't it have a new hand from the one that manages the main game?